tests/kwio_test.sh

```diff
@@ -237,4 +237,24 @@ function test_ask_yN()
   assert_equals_helper "$assert_equals_message" "$LINENO" '0' "$output"
 }
 
+function test_load_module_text()
+{
+  path="$PWD/tests/samples/load_module_text_test_samples/"
+
+  load_module_text "$path/file_correct" > /dev/null
```

Comment:
Why do we need `/dev/null` here? I expected no output in the "good case".

Reply:
You are right!